ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverManagerImpl.java

```diff
@@ -19,19 +19,31 @@
     Session session;
 
     @Override
-    public Cluster registerCluster(String clName, String contactPoint, int connections, int port) {
+    public Cluster registerCluster(String clName, String contactPoint, int connections, int port, String username, String password) {
     
         PoolingOptions poolingOpts = new PoolingOptions()
                                      .setConnectionsPerHost(HostDistance.LOCAL, connections, connections)
                                      .setMaxRequestsPerConnection(HostDistance.LOCAL, 32768);
     
-        cluster = Cluster.builder()
-                .withClusterName(clName)
-                .addContactPoint(contactPoint)
-                .withPoolingOptions(poolingOpts)
-                .withPort(port)
-                .withLoadBalancingPolicy( new TokenAwarePolicy( new RoundRobinPolicy() ) )
-                .build();
+
```

Comment:
Can you simplify this here? Let the cluster builder be common and append credentials needed. Something like,


 ```
Cluster.Builder clusterBuilder = Cluster.builder()
                  .withClusterName(clName)
                .addContactPoint(contactPoint)
                   .withPoolingOptions(poolingOpts)
                    .withPort(port)
                    .withLoadBalancingPolicy( new TokenAwarePolicy( new RoundRobinPolicy() ) );
 if ((username != null) && (password != null)) {
clusterBuilder.withCredentials(username, password);
}```

Reply:
Added this changes...